boofuzz/helpers.py

```diff
@@ -458,15 +455,15 @@ def get_boofuzz_version(boofuzz_class):
         for line in search:
             # line = line.rstrip()  # remove '\n' at end of line
             if line.find("__version__ = ") != -1:
-                version = 'v' + re.search("\'(.*?)\'", line).group(1)
+                version = 'v' + re.search(r"\'(.*?)\'", line).group(1)
```

Comment:
It does indeed when using print. However the functions are working just as they did before.
The problem without treating those strings as raw is, that e.g `\s` is an invalid escape sequence (used in setup.py).
So we either define them as raw strings, or escape the characters like `\\s`.

In this exact case raw flag isn't necessary, I just added it because it's a regex and for consistency.
We could of course change the function in setup.py to work like this one, so the regex doesn't contain any invalid escape sequences anymore.

Reply:
Ah, right, since it's a regex, `\'` matches a literal `'` character, so there is no change in how the regex is parsed.

For simplicity, it would make sense to take out the two backslashes here. It won't change the meaning but will simplify the regex.